src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+  }
+
+  @for $starCount from 1 to 5 {
+    &--#{$starCount} &__star:nth-child(-n+#{$starCount}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  & &__star:nth-child(5) {
+    background-image: url(../images/star.svg);
+  }
```

Comment:
you're sharing an issue with `class="stars stars--5"` with your Brothers - let's have it fixed.

*this class won't show 5 active stars